packages/apps/src/SideBar/index.tsx

```diff
@@ -120,7 +124,7 @@ class SideBar extends React.PureComponent<Props, State> {
     const { api, isApiReady } = this.props;
 
     return (
-      <div className='apps--SideBar-logo'>
+      <div className='apps--SideBar-logo' onClick={this.openModal('network')}>
```

Comment:
```suggestion
      <div
        className='apps--SideBar-logo'
        onClick={this.openModal('network')}
      >
```

Small request - where we have multiple props, split them to one per line.

Reply:
As per the `closeModal`, rather make this a `openNetworkModal` or align with what we have in the rest of the codebase with a `toggleNetworkModal` function. (Preferred)